client/src/App.js

```diff
@@ -22,7 +24,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path='/' render={ (props) => <MovieList {...props} addToSavedList={(movie) => {this.addToSavedList(movie)}}/> } />
```

Comment:
```suggestion
        <Route exact path='/' render={ (props) => <MovieList {...props} addToSavedList={this.addToSavedList}}/> } />
```
No need for a callback.